adminSiteServer/adminRouter.tsx

```diff
@@ -344,6 +344,26 @@ getPlainRouteWithROTransaction(
         )
     }
 )
+
+getPlainRouteWithROTransaction(
+    adminRouter,
+    "/charts/:id/preview",
+    async (req, res, trx) => {
+        const chartId = expectInt(req.params.id)
+        const chart = await getChartConfigById(trx, chartId).catch(
```

Comment:
Why do we hide the errors with the catch? Won't this mean we get `undefined` and thus 404 for cases like DB timeouts, OOM and such, which would make debugging quite confusing? I think we can probably just omit the `catch` completely and the error should get into Sentry.

Reply:
yeah, i just cop-pasted it from below without thinking much about it

Reply:
removed it 👍🏻

Reply:
Thanks for also fixing the exisintg code!

Reply:
> Thanks for also fixing the existing code!

Ah, for the existing code this was actually load-bearing, in a way 🙃 
That's because `getChartConfigBySlug` will throw when it doesn't find a config (which is a... choice), and so in that case the route will fail early, without a chance to look for MDDs under that slug / catalog path.

Reply:
bringing it back with a note...